java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -1086,6 +1088,7 @@ void testWindowStatic() {
   }
 
   @Test
+  @Disabled
```

Comment:
Would be nice to have a comment explaining why tests are marked disabled, otherwise future maintainers are left wondering under what criteria it can be enabled or why it's here at all, e.g.: `@Disabled("cudf bug with negative timestamps, see https://...")`

Reply:
This wasn't supposed to go in.